validation/vortex_street_2d/vortex_street_2d.jl

```diff
@@ -0,0 +1,236 @@
+                                                      AdamiPressureExtrapolation(),
+                                                      state_equation=state_equation,
+                                                      viscosity=viscosity,
+                                                      smoothing_kernel, smoothing_length)
+
+boundary_system_cylinder = BoundarySPHSystem(cylinder, boundary_model_cylinder)
+
+# ==========================================================================================
+# ==== Postprocessing
+circle = SphereShape(particle_spacing, (cylinder_diameter + particle_spacing) / 2,
+                     cylinder_center, fluid_density, n_layers=1,
+                     sphere_type=RoundSphere())
+
+# Points for pressure interpolation, located at the wall interface.
+const data_points = reinterpret(reshape, SVector{ndims(circle), eltype(circle)},
+                                circle.coordinates)
+const center = SVector(cylinder_center)
+
+calculate_lift_force(system, v_ode, u_ode, semi, t) = nothing
+function calculate_lift_force(system::TrixiParticles.FluidSystem, v_ode, u_ode, semi, t)
+    force = zero(first(data_points))
+
+    for point in data_points
+        values = interpolate_points(point, semi, system, v_ode, u_ode; cut_off_bnd=false,
+                                    clip_negative_pressure=false)
+
+        # F = ∑ -p_i * A_i * n_i
+        force -= values.pressure * particle_spacing .*
+                 TrixiParticles.normalize(point - center)
+    end
+
+    return 2 * force[2] / (fluid_density * prescribed_velocity^2 * cylinder_diameter)
+end
+
+calculate_drag_force(system, v_ode, u_ode, semi, t) = nothing
+function calculate_drag_force(system::TrixiParticles.FluidSystem, v_ode, u_ode, semi, t)
+    force = zero(first(data_points))
+
+    for point in data_points
+        values = interpolate_points(point, semi, system, v_ode, u_ode; cut_off_bnd=false,
+                                    clip_negative_pressure=false)
+
+        # F = ∑ -p_i * A_i * n_i
+        force -= values.pressure * particle_spacing .*
+                 TrixiParticles.normalize(point - center)
+    end
+
+    return 2 * force[1] / (fluid_density * prescribed_velocity^2 * cylinder_diameter)
+end
+
+# ==========================================================================================
+# ==== Simulation
+semi = Semidiscretization(fluid_system, open_boundary_in, open_boundary_out,
+                          boundary_system_wall, boundary_system_cylinder)
+
+ode = semidiscretize(semi, tspan)
+
+info_callback = InfoCallback(interval=100)
+
+output_directory = "out_vortex_street_dp_$(factor_D)D_c_$(sound_speed)_h_factor_$(h_factor)_" *
+                   TrixiParticles.type2string(smoothing_kernel)
+if write_to_vtk
+    saving_callback = SolutionSavingCallback(; dt=0.02, prefix="", output_directory)
+else
+    saving_callback = nothing
+end
```

Comment:
Why? You can just set `saving_callback=nothing` in `trixi_include`. This just adds unnecessary complexity to the already complex example file.